src/software/handheld_controller/controller_primitive_generator_test.cpp

```diff
@@ -28,3 +31,67 @@ TEST(ControllerPrimitiveGeneratorTest, test_create_direct_velocity)
               0.5);
     EXPECT_EQ(direct_velocity_primitive->direct_control().dribbler_speed_rpm(), 200);
 }
+
+TEST(ControllerPrimitiveGeneratorTest, test_create_primitive_controller_input)
+{
+    ControllerInput input = ControllerInput();
```

Comment:
can we add a brief comment at the top of every test to explain what's happening?

Reply:
Ok, will do